holidayextras/brands
# Creation of Alton Towers, Chessington and Thorpe Park Brands

#### What does this PR do? (please provide any background)
- Creates 2 new brands in our brand repo, one for Thorpe Park and one for Alton Towers
- Modify's the exiting Chessington brand to work a little better.

#### What tests does this PR have?
None

#### How can this be tested?
This is linked to another PR - 
Ensure you are on this branch of brands
Clone `pattren-library` and switch to branch `alton`
`npm link ../brands/`
`npm start` on the pattern library
go to `pages` and select `shortbreaks payment`
Now switch brands, play with your screen size and test.

#### Screenshots / Screencast
<img width="360" alt="screen_shot_2016-02-05_at_17 20 11_720" src="https://cloud.githubusercontent.com/assets/778942/12983623/f31a009e-d0e1-11e5-90d0-ebf95979836f.png">
<img width="360" alt="screen_shot_2016-02-05_at_17 22 08_720" src="https://cloud.githubusercontent.com/assets/778942/12983622/f317471e-d0e1-11e5-8ef3-d6b865ffb425.png">



#### What gif best describes how you feel about this work?
![YEP](https://media.giphy.com/media/3oEduQotroJjxc73SU/giphy.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
